<?php

$ex = new Exception('Сообщение об исключении');

echo $ex->getMessage();

throw  $ex;



